keyboards/mawaeg/sharkropad/config.h

```diff
@@ -0,0 +1,22 @@
+/* Copyright 2025 @mawaeg
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#define STM32_FLASH_SECTORS_PER_BANK 128
+
+#define I2C1_SCL_PIN A9
+#define I2C1_SDA_PIN A10
+
+#define WS2812_BITBANG_NOP_FUDGE 0.4
```

Comment:
Would it be possible to use the WS2812 PWM driver, instead of the bitbang driver? The B3 pin you're using looks like it can use `TIM2_CH2`, although pretty sure you'll need to use a different system timer (`STM32_ST_USE_TIMER x`). See [WS2812 Documentation](https://docs.qmk.fm/drivers/ws2812).
```suggestion
```